kristan/src/test/java/com/example/springboot/RepositoryTest.java

```diff
@@ -0,0 +1,22 @@
+package com.example.springboot;
+import com.example.springboot.repository.Students;
+import com.example.springboot.repository.Programs;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.util.Assert;
+
+
+public class RepositoryTest {
+
+    @Test
+    void getAllProgramsTest() {
+        Programs initRepo = new Programs();
+        Assert.isTrue(initRepo.getAllPrograms().size() == 3, "Programs Repo Test");
+    }
+
+    @Test
+    void getAllStudentsTest() {
+        Students initRepo = new Students();
+        Assert.isTrue(initRepo.getAllStudents().size() == 3, "Students Repo Test");
+    }
```

Comment:
Good Day, JUnit test is very helpful, it allows us to check and identify quickly if there is something missing on our data. Thanks for introducing this method to me.